src/main/kotlin/racing/domain/CarFactory.kt

```diff
@@ -0,0 +1,16 @@
+package racing.domain
+
+class CarFactory(
+    private val carMaxPower: Int = DEFAULT_MAX_POWER,
+    private val powerStrategy: PowerStrategy = RandomPowerStrategy(carMaxPower)
+) {
+    fun createCars(carNames: List<String>): RacingCars {
+        return RacingCars(carNames.map { name ->
+            Car(name = name, powerStrategy = powerStrategy)
+        })
```

Comment:
```suggestion
        return RacingCars(carNames, powerStrategy)
```

`CarFactory` 가 `RacingCars`를 넘어 `Car` 의 생성까지 관여하는 것은 너무 많은것을 알고 있는것 같습니다. `Car` 객체 생성을 `RacingCars` 에게 위임해 버리는건 어떨까요?

Reply:
RacingCars 가 Car 를 생성하도록 만들기 위해 RacingCars 가 powerStrategy 를 아는것 보단 
CarFactory 에서는 List<Car> 로만 생성하고 그걸 RacingCars 로 감싸는건 외부에서 하는게 더 좋지 않을까 라는 의견이 있습니다.

Reply:
👌